chia/util/keychain.py

```diff
@@ -284,6 +286,7 @@ def _get_key_data(self, index: int, include_secrets: bool = True) -> KeyData:
         return KeyData(
             fingerprint=public_key.get_fingerprint(),
             public_key=public_key,
+            label=self.keyring_wrapper.get_label(public_key.get_fingerprint()),
```

Comment:
nit: assign fingerprint to local variable to avoid extra call to `get_fingerprint()`?

Reply:
See 2fdbf835db7c6b7345de505627b28c9eaf693860